telegram/ext/_application.py

```diff
@@ -1058,6 +1075,17 @@ def add_handler(self, handler: BaseHandler[Any, CCT], group: int = DEFAULT_GROUP
             might lead to race conditions and undesired behavior. In particular, current
             conversation states may be overridden by the loaded data.
 
+        .. seealso:: `Your First Bot <https://github.com/\
+            python-telegram-bot/python-telegram-bot/wiki/Extensions-–-Your-first-Bot>`_,
+            `Architecture Overview <https://github.com/\
+            python-telegram-bot/python-telegram-bot/wiki/Architecture>`_,
+            `Builder Pattern <https://github.com/\
+            python-telegram-bot/python-telegram-bot/wiki/Builder-Pattern>`_,
+            `Storing Bot, User and Chat Related Data <https://github.com/\
+            python-telegram-bot/python-telegram-bot/wiki/Storing-bot%2C-user-and-\
+            chat-related-data>`_,
+            :any:`Echobot Example <examples.echobot>`
+
```

Comment:
builder pattern and stronig data are rather unerlated to this IMO. And the tutorials directly links to echobot, so I'd also skip that :)

Reply:
deleting links to Builder Pattern, Storing Bot..., echobot example